src/slacko.ml

```diff
@@ -834,7 +841,11 @@ let string_of_presence = function
   | Away -> "away"
 
 (* Slacko API helper methods *)
-let token_of_string = identity
+let make_session ?(base_url=default_base_url) token = {
```

Comment:
Any of those work, and I don't have particularly strong opinions on which. `start_session` seems slightly more sensible than the others, so I'll change to that.